DEV_2/DEV_2UnitTest/InputDataTests.cs

```diff
@@ -0,0 +1,104 @@
+﻿
+namespace DEV_2UnitTest
+{
+    using System;
+
+    using DEV_2;
+
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+    /// <summary>
+    /// The input data tests.
+    /// </summary>
+    [TestClass]
+    public class InputDataTests
```

Comment:
you can make main so trivial that it doesn't need a test by delegating all it's functions to some non-static class that can be tested.